draft-vvv-webtransport-http3.md

```diff
@@ -157,10 +157,61 @@ is closed.
 
 ## Establishing a Transport-Capable HTTP/3 Connection
 
+In WebTransport, an *exclusive mode* is a state of HTTP/3 connection where the
+connection is expected to have exactly one WebTransport session, and no other
+HTTP requests.
+
 In order to indicate support for WebTransport, both the client and the server
-MUST send a SETTINGS_ENABLE_WEBTRANSPORT value set to "1" in their SETTINGS
-frame.  Endpoints MUST NOT use any WebTransport-related functionality unless
-the parameter has been negotiated.
+MUST send a SETTINGS_ENABLE_WEBTRANSPORT setting.  The value of
```

Comment:
We need to state somewhere that if SETTINGS_ENABLE_WEBTRANSPORT is not sent, its value is implicitly set to the default of 0.

Reply:
Fixed.